app/controllers/posts_controller.rb

```diff
@@ -8,4 +8,20 @@ def show
     @user = User.find(params[:user_id])
     @post = Post.find(params[:id])
   end
```

Comment:
- [ ] There is a button on the user show page, when the button is clicked, we expect to have all the posts for that user. Currently, there is an error because you are not passing a user ID as a param when fetching posts in the post index action. Kindly fix this issue.

![posts](https://user-images.githubusercontent.com/53124548/219845509-5470289f-ede2-4c2d-bc9c-7866d7d572e1.png)